docs/foutmeldingen.json

```diff
@@ -0,0 +1,38 @@
+{
+ "paramsRequired": "Geef tenminste één parameter op.",
```

Comment:
Ik zou dit als volgt opbouwen:

``` json
[
  { "code": "paramsRequired", "titel": "Minimale combinatie van parameters moet worden opgegeven."},
  { "code": "paramsCombination", "titel": "Minimale combinatie van parameters moet worden opgegeven."},
]
```
Dit heeft als voordeel dat het makkelijk kan worden uitgebreid met bijv. fout omschrijving, status code.

Reply:
maar heeft dacht ik als nadeel dat de lijst moet worden geïtereerd om bij een gegeven code de gevraagde title te vinden.
Voorzie je dat we deze lijst gaan uitbreiden met bijvoorbeeld fout omschrijving of status code?

Reply:
M.b.v. json path hoeft je niet te itereren. In Java en C# zijn er libraries waarmee je dit ook op soortgelijk manier kan doen.

Ik kan mij heel goed voorstellen dat reason(s) ook wordt toegevoegd.

Reply:
ik heb de structuur omgezet zoals je hebt voorgesteld